### PR TITLE
(DOCSP-26079): Update Dart Standalone/Flutter SDKs

### PR DESCRIPTION
**Pull request by mongodben**
## Pull Request Info

Document how to update Dart Standalone/Flutter SDKs.

### Jira

- https://jira.mongodb.org/browse/DOCSP-26079

### Staged Changes

- [Install (Flutter), Update Package Version section](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-26079/sdk/flutter/install#update-package-version)

### Reminder Checklist

If your PR modifies the docs, you might need to also update some corresponding
pages. Check if completed or N/A.

- [x] Create Jira ticket for corresponding docs-app-services update(s), if any
- [x] Checked/updated Admin API
- [x] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk/flutter/install.txt [hunk 1]
```diff
@@ -157,3 +157,76 @@ except where otherwise noted.
...
+
+         .. code-block::
+
+            flutter pub upgrade realm
+
+      #. Regenerate your Realm Object models.
+
+         .. code-block::
+
+            dart run realm_dart generate
```
Comment (desistefanova) [inline source/sdk/flutter/install.txt:192]:
```suggestion
            flutter pub run realm generate
```

Comment (nielsenko) [reply source/sdk/flutter/install.txt:192]: sorry no.. you could run:
```sh
dart run realm generate
```
as well

Comment (nielsenko) [reply source/sdk/flutter/install.txt:192]: .. anyway better go with @desistefanova suggestion..

Comment (mongodben) [reply source/sdk/flutter/install.txt:192]: thanks! this was def some bad copy pasting on my part

File: source/sdk/flutter/install.txt [hunk 2]
```diff
@@ -157,3 +157,76 @@ except where otherwise noted.
       .. code-block:: dart
 
         import 'package:realm_dart/realm.dart';
+
+.. _flutter-update-package:
+
+Update Package Version
+----------------------
+
+If you would like to use a version of the Flutter SDK or Dart Standalone SDK
```
Comment (dacharyc) [inline source/sdk/flutter/install.txt:166]: The wording on this just feels a little awkward to me. Could we say something simpler like:

"To change the version of the Flutter SDK or Dart Standalone SDK in your project..."

**Review by dacharyc — APPROVED** [review]
Overall, LGTM. I just wonder if we can simplify the opening sentence a little, but not a blocker.

File: source/sdk/flutter/install.txt [hunk 3]
```diff
@@ -157,3 +157,76 @@ except where otherwise noted.
...
+different from the one currently in your project, follow these steps.
```
Comment (mongodben) [inline source/sdk/flutter/install.txt:167]:
```suggestion
To change the version of the Flutter SDK or Dart Standalone SDK
in your project, follow these steps.
```

**Pull request merged**